src/entries/popup/hooks/useInfiniteTransactionList.ts

```diff
@@ -53,9 +56,20 @@ export const useInfiniteTransactionList = ({
   const { testnetMode } = useTestnetModeStore();
   const { chains } = useUserChains();
   const userChainIds = chains.map(({ id }) => id);
-  const supportedChainIds = useSupportedChains({ testnets: testnetMode })
-    .map(({ id }) => id)
-    .filter((id) => userChainIds.includes(id));
+  const supportedChains = useSupportedChains({ testnets: testnetMode });
+
+  // stable reference
+  const supportedChainIds = useMemo(
+    () =>
+      supportedChains
+        .map(({ id }) => id)
+        .filter((id) => userChainIds.includes(id)),
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    supportedChains
+      .map(({ id }) => id)
+      .filter((id) => userChainIds.includes(id)),
+  );
```

Comment:
supportedChains actually could change when testnet mode is activated, chain list changes (network added/disabled)